azurelinuxagent/common/utils/archive.py

```diff
@@ -43,7 +43,8 @@
 _CACHE_PATTERNS = [
     re.compile(r"^(.*)\.(\d+)\.(agentsManifest)$", re.IGNORECASE),
     re.compile(r"^(.*)\.(\d+)\.(manifest\.xml)$", re.IGNORECASE),
-    re.compile(r"^(.*)\.(\d+)\.(xml)$", re.IGNORECASE)
+    re.compile(r"^(.*)\.(\d+)\.(xml)$", re.IGNORECASE),
```

Comment:
what files is this one trying to capture? thanks

Reply:
This pattern was already present as a part of previous code. Digging further its matches files such as 'GoalState.0.xml', 'Microsoft.Azure.Extensions.CustomScript.0.xml' .These files will be added to the list of files that needs to be archived to history

Reply:
ok